python/pyspark/mllib/tree.py

```diff
@@ -90,9 +92,11 @@ def predict(self, x):
         else:
             return self.call("predict", _convert_to_vector(x))
 
+    @property
```

Comment:
Do you think we should we should support both, with the method being a wrapper around the property?